src/components/modals/FlipInputModal.js

```diff
@@ -79,6 +80,8 @@ class FlipInputModalComponent extends React.PureComponent<Props, State> {
     updateTransactionAmount(nativeAmount, exchangeAmount, walletId, currencyCode)
   }
 
+  handleAmountChangeError = (error?: string) => this.setState({ errorMessage: error })
```

Comment:
change the param name to "errorMessage" for 2 reasons.
1. Error usually represent an error object and not the message.
2. By changing the param name, the code would look like this:
`(errorMessage?: string) => this.setState({ errorMessage })`